.expeditor/build_gems.ps1

```diff
@@ -84,32 +85,10 @@ Write-Output "Hab thinks it installed my 64-bit dlls here : $x64"
 Test-Path -Path $x64
 Write-Output "`r"
 
-Write-Output "--- :construction: Building 32-bit PowerShell DLL's"
```

Comment:
Are we no longer shipping a 32 bit chef client for windows? If so, should you also delete all the other x86 crap (using this as a term of endearment).

Reply:
We aren't but do you have additional suggestions where to delete? Not familiar with all the places things can hide in a VS project.